modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartJaguarClientCodegen.java

```diff
@@ -142,6 +147,13 @@ public void processOpts() {
             additionalProperties.put(PUB_NAME, pubName);
         }
 
```

Comment:
@athornz what about setting clientName here using StringUtils methods?

That way we don't need to introduce another option

Reply:
Great suggestion, works really well. I've pushed the changes now.